src/bundlers/go/goBundler.ts

```diff
@@ -227,22 +233,24 @@ export class GoBundler implements BundlerInterface {
 
     async #compile(folderPath: string) {
         // Compile the Go code locally
-        const getDependencyResult = spawnSync(
-            "go",
-            ["get", "github.com/aws/aws-lambda-go/lambda"],
-            {
+        const dependencies = [
+            "github.com/aws/aws-lambda-go/lambda",
+            "github.com/Genez-io/genezio_types",
+        ];
+        for (const dependency of dependencies) {
+            const getDependencyResult = spawnSync("go", ["get", dependency], {
```

Comment:
Please replace `spawnSync` with `execa`. We want to slowly roll out of using `child_process` because it has a lot of flaws. `execa` is a wrapper over `child_process` that improves Windows support. https://www.npmjs.com/package/execa

Please also do this in `localGoBundler.ts`

Reply:
Here is an example https://github.com/Genez-io/genezio/blob/dev/src/packageManagers/pnpm.ts#L2

Reply:
wow, thats looks very nice. thanks!